core/src/main/java/org/apache/calcite/rel/core/JoinRelType.java

```diff
@@ -25,6 +25,7 @@
   /**
    * Inner join.
    */
+
```

Comment:
I think that this change is unnecessary.

Reply:
thanks